Assets/Scripts/Grid/Grid.cs

```diff
@@ -40,5 +40,6 @@ public class Grid : IGrid {
             _numTilesX = System.Math.Max(1, gridData.NumTilesX);
             _numTilesY = System.Math.Max(1, gridData.NumTilesY);
         }
+        
```

Comment:
Please don't commit this extra space.

Reply:
Yeah, this was with the diff tool, something happened. I will be more careful next time.